tests/sdk_test/sdk_test.cpp

```diff
@@ -308,7 +312,9 @@ void MegaChatApiTest::SetUp()
 #ifndef KARERE_DISABLE_WEBRTC
         mCallReceived[i] = false;
         mCallAnswered[i] = false;
-        mCallEmisorId[i] = MEGACHAT_INVALID_HANDLE;
+        mChatIdRingInCall[i] = MEGACHAT_INVALID_HANDLE;
+        mTerminationLocal[i]= false;
+        mTerminationCode[i] = mInvalidTermCode;
```

Comment:
Review the state variables declared when WebRTC. There are several vars that are not here, so the initialization is missing. More concretely, there are 11 variables for this matters, and only 5 are initialized here :(